internal-dev/cheminf-ext.owl

```diff
@@ -82,7 +74,13 @@
         <rdfs:label xml:lang="en">cdk descriptors</rdfs:label>
     </owl:Class>
     
+    <!-- http://purl.enanomapper.org/onto/ENM_1200002 -->
 
+    <owl:Class rdf:about="http://purl.enanomapper.org/onto/ENM_1200002">
+        <rdfs:subClassOf rdf:resource="http://semanticscience.org/resource/CHEMINF_000464"/>
+        <rdfs:isDefinedBy>PMCIDr is a unique identifier for publications archived in PubMed Central, which may also be indexed in PubMed. </rdfs:isDefinedBy>
```

Comment:
The `r` after `PMCID` in the description is not intented, I guess?

Reply:
Sorry didn't see that, must be a typo. I'll fix it right away!